dataactcore/interfaces/function_bag.py

```diff
@@ -296,7 +296,7 @@ def mark_job_status(job_id, status_name):
 
     # if status is changed to finished for the first time, check dependencies
     # and add to the job queue as necessary
-    if old_status != 'finished' and status_name == 'finished':
+    if old_status != 'finished' and status_name == 'finished' and not skip_check:
```

Comment:
Right, but where do the validation jobs get marked as finished if they're never kicked off?